reports/reports/m2-reflection.md

```diff
@@ -0,0 +1,33 @@
+# Reflection
+
+## Implementation Status
+
+Our dashboard, "The Future of Democracy and American Elections," aimed to offer an in-depth analysis of public sentiment towards American politics in August 2018. We planned to include a variety of interactive elements such as demographic filters (age, race, education level, political ideology), and visualization components (bar charts, donut charts, tabulated data) to enable users to explore the dataset effectively.
+
+We successfully effectively presented complex data through visualizations that cater to various aspects of American political opinions and demographics. The use of interactive sliders and demographic dropdown filters  allows for dynamic exploration of the data, and the color-coded bar and pie charts provide at-a-glance insights into political affiliations and opinions on key issues like the likelihood of war and election fairness.
+
+Current limitations include a single-screen layout that may hamper quick interpretation, an inability to directly compare demographic groups side by side, and the absence of a summary banner to communicate overarching narratives or conclusions. Additionally, lacking hover-over details on data points restricts the immediate availability of specific information. 
+
+## Deviations from the Proposal
+
+Instead of implementing a static tabulated data view, we decided to incorporate an interactive table that users can sort and filter. This change was made to enhance the dashboard's interactivity and user engagement, allowing for a more personalized exploration of the data. We found this approach more effective in providing users with the specific insights they are interested in, rather than a fixed data presentation.
+
+## Current Strengths
+
+The dashboard excels in offering a customizable and interactive exploration of political sentiment data, particularly through its demographic filters and dynamic bar charts. Users can effectively dissect the data according to specific demographics of interest, making the tool valuable for in-depth analysis.
```

Comment:
- can we mention the color choice --> that we went for the aesthetic and not just
- can we also mention the rationale behind choosing stacked bar charts, donut chart instead of a better visualisation ( it is easier for most people to interpret etc) 
- can we also mention that we switched from single select drop downs to  multi-select to allow better comparisons and more customizability
- age slider - why did we use that instead of grouped age buckets ?
- table was used so it is easier for general public to understand and interpret 
- the dashboard is divided into general correlations and donald trump focused info so the users can get a better idea of the general demographic views compared to their views regarding donald trump